packages/web/src/scss/components/Toast/_Toast.scss

```diff
@@ -134,27 +134,27 @@ $_queue-collapsed-visible-limit: 2; // 10.
 }
 
 // 9.
-.Toast--collapsible > .Toast__queue > :nth-last-child(-n + #{theme.$queue-collapse-after}) {
+.Toast--collapsible > .Toast__queue > :where(:nth-last-child(-n + #{theme.$queue-collapse-after})) {
```

Comment:
👍🏻 for the specificity fix.

Will you please drop a comment explaining why `where` is used? I think inserting a new "11" between current 11 and 12 is a good place.